script/fetchneedles

```diff
@@ -1,23 +1,32 @@
 #!/bin/sh -e
-[ "$1" = "-h" ] || [ "$1" = "--help" ] && echo "Get openSUSE tests and needles from https://github.com/os-autoinst/os-autoinst-distri-opensuse" && exit
-
 : "${dbuser:="geekotest"}"
 : "${dbgroup:="www"}"
 
+: "${dist_name:=${dist:-"OpenSUSE"}}" # the display name, for the help message
```

Comment:
It's actually "openSUSE" with a lowercase leading character.
It's only for the help text but I am sure you understand that we should be consistent about the project's names :)

Reply:
Fixed that with an updated commit